include/internal/stack_of_comp_methods.h.in

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+{-
+use OpenSSL::stackhash qw(generate_stack_macros generate_const_stack_macros);
+-}
+
+#ifndef OPENSSL_STACK_OF_COMP_METHODS_H
+
+{-
+    generate_stack_macros("OSSL_COMP");
```

Comment:
IIRC I don't think this is necessary for internal stacks. I think you can just do `DEFINE_STACK_OF(OSSL_COMP)` directly in cryptlib.h and remove this header entirely.